src/Common/hooks/useFilters.tsx

```diff
@@ -18,15 +19,25 @@ interface FilterBadgeProps {
  * A custom hook wrapped around raviger's `useQueryParams` hook to ease handling
  * of pagination and filters.
  */
-export default function useFilters({ limit = 14 }: { limit?: number }) {
+export default function useFilters({
+  limit = 14,
+  cacheBlacklist = [],
+}: {
+  limit?: number;
+  cacheBlacklist?: string[];
+}) {
   const { t } = useTranslation();
   const { kasp_string } = useConfig();
   const hasPagination = limit > 0;
   const [showFilters, setShowFilters] = useState(false);
   const [qParams, setQueryParams] = useQueryParams();
+  const [primaryCacheBlacklist, setPrimaryCacheBlacklist] =
+    useState(cacheBlacklist);
```

Comment:
Why is a state required for this? We could simply do:

```suggestion
  const cacheBlacklist = FILTERS_CACHE_BLACKLIST.concat(cacheBlacklist);
```

Reply:
needed to use the state to get data from the updateQuery 
due to PhoneNumberFormField else directly using the useFilter params works

Reply:
It'd be nice to avoid complications and make the code readable and straight forward.
If there's an issue with how phone number updates the query, that should be fixed.

But I don't think that's an issue since even if you had added the phone number field's key in the global list, it'd work the same way without these complications right?